liferay-echo/liferay-portal
# LPS-184940 Refine the process of importing page templates

**IMPORTANT!** This is a follow up PR of https://github.com/liferay-echo/liferay-portal/pull/12141

Motivation
=======
We want to improve the process of importing page templates.
[Link to story or ticket](https://issues.liferay.com/browse/LPS-184940)

Proposed Solution
========
We have created a new modal for importing in React so that we can customize better its UX. We have also move the warning messages to the view.jsp so that they are visible after closing the modal.
There will be follow up PRs changing the alert messages and the importation of page templates.

Steps to Verify:
----------------
1. Add `feature.flag.LPS-174939=true` to your portal-ext.properties
2. Go to Design -> Page Templates -> Page Templates tab and in the three dots menu click on Import.
3. Check that the modal width is 600px.
4. Check that there is a paragraph before the input with the text: “Select a Zip File containing one or multiple entries.”.
5. Check that the "Select file" label has been replaced with with "File".
6. The text "Read more" text should be now "Read More about Exporting and Importing Page Templates".
7. Check that the "Read more" link is pointing to:
https://learn.liferay.com/dxp/latest/en/site-building/creating-pages/adding-pages/exporting-and-importing-page-templates.html
8. The "Overwrite Existing Entries" label should not be bold.
9. The import button should be disabled until a zip file is selected.
10. Try importing page templates with no errors and with some errors. Check that the success, error and warning messages appear outside the modal and after the modal has been closed.
11. Check that everything works without the Feature Flag
